src-docs/src/views/tool_tip/tool_tip.js

```diff
@@ -27,7 +27,7 @@ export default () => (
       <p>
         This tooltip appears on the{' '}
         <OuiToolTip position="top" content="Here is some tooltip text">
-          <OuiLink href="#">top</OuiLink>
+          <OuiLink href="https://oui.opensearch.org/latest/">top</OuiLink>
```

Comment:
<img width="600" alt="Screenshot 2023-04-18 at 09 35 04" src="https://user-images.githubusercontent.com/17855243/232664463-e960f3a1-c9a6-49e4-b337-a75a3793bf4d.png">

Reply:
These should not be links, the # is on purpose to show a tooltip only.

Reply:
Reverted changes

<img width="600" alt="Screenshot 2023-04-19 at 10 12 35" src="https://user-images.githubusercontent.com/17855243/232965259-e0603e76-e05e-4bb3-bc65-d93d063f5f53.png">